tests/runner/test_runner.py

```diff
@@ -31,21 +32,33 @@
 MODULE_PATH = "tmp_dir"
 RUN_CONFIG_PATH = os.path.join(MODULE_PATH, "run_config.yml")
 CUSTOM_FOLDER_PATH = os.path.join(MODULE_PATH, "custom_nodes")
+CUSTOM_CONFIG_FOLDER_PATH = os.path.join(MODULE_PATH, "configs", PKD_NODE_TYPE)
 PKD_NODE_DIR = os.path.join(MODULE_PATH, PKD_NODE_TYPE)
 CONFIG_UPDATES_CLI = "{'input.live': {'resize':{'do_resizing':True}}}"
 
 
 class MockedNode(AbstractNode):
     def __init__(self, config):
-        super().__init__(config, node_path=PKD_NODE)
+        super().__init__(config, node_path=PKD_NODE, pkdbasedir=MODULE_PATH)
 
     def run(self, inputs):
         output = {}
-        for idx in range(len(self._outputs)):
-            output[self._outputs[idx]] = "test_output_" + str(idx)
+        for idx in range(len(self.outputs)):
+            output[self.outputs[idx]] = "test_output_" + str(idx)
 
         return output
 
+def create_node_config(config_dir, node_name):
+    
+    config_text = {"root": None,
+                   "input": ["source"],
+                   "output": ["end"]}
```

Comment:
These 2 types are deprecated and we shouldn't be using them. Suggest using ["none"] and ["pipeline_end"] respectively instead.

Reply:
yeps, updated the changes to "none" and "pipeline_end" respectively.